main.tf

```diff
@@ -68,7 +68,7 @@ resource "aws_iam_role_policy" "traefik_policy" {
 }
 
 resource "aws_iam_role" "ecs_role" {
-  name               = "ecs_role"
+  name               = "${var.ecs_cluster_name}_ecs_role"
```

Comment:
Should we call this `${var.ecs_cluster_name}_traefik` so that it won't conflict with other role names?